provider/cluster_rosa_classic_resource.go

```diff
@@ -22,19 +22,23 @@ import (
 	"encoding/hex"
 	"fmt"
 	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"net/http"
+	"net/url"
+	"strings"
+
+	semver "github.com/hashicorp/go-version"
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
 	"github.com/hashicorp/terraform-plugin-framework/types"
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 	"github.com/openshift-online/ocm-sdk-go/logging"
-	"net/http"
-	"net/url"
-	"strings"
 )
 
 const (
-	awsCloudProvider = "aws"
-	rosaProduct      = "rosa"
+	awsCloudProvider  = "aws"
+	rosaProduct       = "rosa"
+	serviceAccountFmt = "system:serviceaccount:%s:%s"
```

Comment:
Its used from `rosa_operator_roles_data_source.go` so why is it defined here?

Reply:
I left it here by a mistake - moved it to be in `rosa_operator_roles_data_source.go`